### DISCUSSION TITLE
使用 stable-diffusion-webui-docker endpoint 应该填写什么地址？

### DISCUSSION THREAD

**Discussion by StringKe** [koishijs/novelai-bot#147] [Q&A] [answered]
https://github.com/AbdBarho/stable-diffusion-webui-docker

使用 Docker 进行部署，`docker compose --profile auto up --build` 启动后查看了 api 文档并没有目前代码中的定义的这些接口。

https://github.com/koishijs/novelai-bot/blob/main/src/index.ts#L247

**Comment by MaikoTan** [comment, marked as answer]
[Clearly you didn't pass `--api` into the launch script](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/API), I haven't used this repo and I don't know what exactly it is. But I think there should be some place you could write command line arguments.
